examples/plugin_example_features.py

```diff
@@ -0,0 +1,20 @@
+
+def compute_some_new_features(rtdc_ds):
+    """The function that does the heavy-lifting"""
+    circ_per_area = rtdc_ds["circ"] / rtdc_ds["area_um"]
+    bright_per_area = rtdc_ds["bright_avg"] / rtdc_ds["area_um"]
+    # returns a dictionary-like object
+    return {"circ_per_area": circ_per_area, "bright_per_area": bright_per_area}
+
+
+info = {
+    "method": compute_some_new_features,
+    "description": "This plugin will compute some possibly useless features",
+    "feature names": ["circ_per_area", "bright_per_area"],
+    "feature labels": ["", ""],
+    "features required": [["circ", "area_um"], ["bright_avg", "area_um"]],
+    "config required": [[], []],
+    "method check required": "",
```

Comment:
This should be a callable. If it is omitted it should default to None (not sure how it is handled in `AncillaryFeature` currently.